cds_sorenson/api.py

```diff
@@ -232,6 +232,15 @@ def _get_quality_preset(subformat_desired_quality, video_aspect_ratio,
     :param video_width: maximum output width for transcoded video
     :returns the transcoding config for a given inputs
     """
+    # For old videos with really low quality, better to encode lowest always
+    _LOWEST_WIDTH = 320
+    _LOWEST_HEIGHT = 240
+    if video_width:
+        video_width = video_width \
+                      if video_width >= _LOWEST_WIDTH else _LOWEST_WIDTH
+    if video_height:
+        video_height = video_height \
+                       if video_height >= _LOWEST_HEIGHT else _LOWEST_HEIGHT
```

Comment:
What about if you move this below, after the `preset_config` retrieved, to have the right aspect ratio?
My fear is that we "break" the aspect ratio guessing by forcing potentially both video and height.